example/lib/pages/playlist_page.dart

```diff
@@ -30,7 +30,7 @@ class _PlaylistPageState extends State<PlaylistPage> {
     );
     _betterPlayerPlaylistConfiguration = BetterPlayerPlaylistConfiguration(
       loopVideos: true,
-      nextVideoDelay: Duration(seconds: 1),
+      nextVideoDelay: Duration(seconds: 3),
```

Comment:
I thought the change were only related to the UI.

Reply:
To me this did seem like a ui component since there is a display component that tells the user how long till the next video.
If 3 seconds is a lot this is honestly one line of code that can easily be modified.